torchvision/csrc/io/decoder/stream.h

```diff
@@ -20,7 +20,8 @@ class Stream {
   virtual ~Stream();
 
   // returns 0 - on success or negative error
-  int openCodec(std::vector<DecoderMetadata>* metadata);
+  // num_threads sets up the codec context for multithreading if needed
+  int openCodec(std::vector<DecoderMetadata>* metadata, int num_threads = 0);
```

Comment:
Can we leave the default to have the same behavior as before (maybe the equivalent to setting it to `num_threads=0` or 1)?

Reply:
yup. num_threads=1 does that, num_threads=0 calls auto, and num_threads >1 sets up custom number of threads; I'll update the defaults to have same behaviour as before for everything other than VideoReader object.